examples/cross.js

```diff
@@ -0,0 +1,69 @@
+var data = require('./data/data.json')
+var streamSift = require('../lib')
+var c = require('colors/safe')
+
+/**
+* $cross
+* $crossOrEqual
+*/
+
+var spec = { input : { $c : 1 } }
```

Comment:
I've been moving away from `spec` and toward `pattern`. `spec` is good but I think `pattern` is even better in context, what do you think? In conversation the word `spec` would still be valid ("the specified pattern", "it did not match the spec" AKA "it did not match the pattern") but just a thought that `pattern` is an improvement for comprehension; a spec can be a lot more than just a pattern.